doc/dependencies.md

```diff
@@ -39,6 +40,7 @@ Some dependencies are not needed in all configurations. The following are some f
 * SQLite is not needed with `--disable-wallet` or `--without-sqlite`.
 * Qt is not needed with `--without-gui`.
 * If the qrencode dependency is absent, QR support won't be added. To force an error when that happens, pass `--with-qrencode`.
+* If the systemtap dependency is absent, USDT support won't compiled in. To force an error when that happens, pass `--enable-ebpf`.
```

Comment:
Is the error supposed to appear during the `./configure` run? In my case, explicitely passing `--enable-ebpf` didn't throw an error when `systemtap-sdt-dev` was still missing, only `ebpf tracing    = no` was displayed on the summary at the end. Tested on Debian bullseye/sid.

Reply:
Thanks. I've tested this and the docs doesn't seem to be correct. I think assumed this to be the case similar to the `--with-qrencode` behavior in the line above.